pyproject.toml

```diff
@@ -14,12 +14,13 @@ packages = [
 openapi-validator = "openapi_to_fastapi.cli:cli_validate_specs"
 
 [tool.poetry.dependencies]
-python = ">=3.7,<4.0.0"
-datamodel-code-generator = "^0.19.0"
-fastapi = ">=0.97.0,<0.99.0"
+python = ">=3.8,<4.0.0"
+datamodel-code-generator = "^0.22.1"
+fastapi = "^0.103.2"
```

Comment:
Rather ensure the other libraries are updated to support latest FastAPI 0.104.1 version and revert this to also support/use that.